images/inference-services/kfserving/run.sh

```diff
@@ -65,9 +106,9 @@ printf "${prediction_url}" > /tekton/results/${TASK_RESULT}
 
 inside_metadata=""
 first_resource=1
-envsubst < /root/template.sh | while read line
+envsubst < /root/template.sh | while read -r line
 do
```

Comment:
you know, instead of this parsing of yaml file using bash, just create the resulting json directly. We already know what  resources are we creating so why pretend we need some universal solution. 
I've already done it for seldon image

Reply:
Makes sense, but can we leave that for another PR? I think it fits better on the PR replacing `envsubs` with `ytt`.

Reply:
sure